packages/run-protocol/src/vaultFactory/types.js

```diff
@@ -85,9 +85,9 @@
  * @property {() => Ratio} getLoanFee
  * @property {() => Promise<PriceQuote>} getCollateralQuote
  * @property {() => Ratio} getInterestRate - The annual interest rate on a loan
- * @property {() => RelativeTime} getChargingPeriod - The period (in seconds) at
+ * @property {() => NatValue} getChargingPeriod - The period (in seconds) at
```

Comment:
these are specifically seconds, not arbitrary RelativeTime. they also can't be relative; they must be positive.